src/scss/notice/main.scss

```diff
@@ -0,0 +1,40 @@
+@import './src/scss/notice/inner';
+@import './src/scss/notice/regular';
+////
+/// @group o-message
+/// @link http://registry.origami.ft.com/components/o-message
+////
+
+/// Notice Messages
+///
+/// Outputs styling for all alert type messages
+/// @example
+/// @include oMessageNotices('my-class');
+///
+/// @param {String} $class - base class name
+/// @param {String} $type - message type ('all', 'notice', 'notice-bleed' or 'notice-inner')
+/// @param {String} $status - message status ('all', 'inform', 'warning', 'warning-light')
+
+@mixin oMessageNotices($class, $type, $status) {
+	@if $type == 'all' {
+		@include oMessageNotice($class);
+		@include oMessageNoticeInner($class);
+
+	} @else if $type == 'notice' or $type == 'notice-bleed' {
+			@include oMessageNotice($class);
+	} @else if $type == 'notice-inner' {
+			@include oMessageNoticeInner($class);
+	} @else {
+		@include _oMessageError("#{$type} is not a notice type, the options are 'notice', 'notice-bleed' or 'notice-inner'");
+	}
+
+	@if $status == 'all' {
+		@each $status in $o-message-notice-status {
+			@include oMessageStatus($status, $class)
+		}
+	} @else if index($o-message-status, $status) != null {
+			@include oMessageStatus($status, $class);
+	} @else {
+		@include _oMessageError("#{$status} is not a notice status, the options are 'inform', 'warning' or 'warning-light'");
+	}
+}
```

Comment:
Totally ignorable but for readability I'd prefer failing early.

Something along these lines.

```
$supported-notices: ('notice', 'notice-bleed');
@if $type != 'all' and not index($supported-notices, $type) {
	// error
}

@if $type == 'notice' or $type == 'all' {
	// notice stuff
}

@if $type == 'notice-inner' or $type == 'all' {
	// notice inner stuff
}
```